rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -278,7 +337,7 @@ private void forEach(InputObjectState uids, Consumer<Uid> consumer) {
                 if (LRALogger.logger.isTraceEnabled()) {
                     LRALogger.logger.tracef(e,
                             "LRARecoverModule: Object store exception %s while unpacking a record of type %s",
-                            e.getMessage(), _transactionType);
+                            e.getMessage(), transactionType);
                 } else if (LRALogger.logger.isInfoEnabled()) {
                     LRALogger.logger.infof(
                             "LRARecoverModule: Object store exception %s while unpacking a record of type: %s",
```

Comment:
The `infof` misses the `transactionType` as the second parameter (there is only one  which is `e.getMessage()`).

Reply:
I will add it thanks.